agent/testsuite-rhel9.sh

```diff
@@ -136,11 +135,25 @@ export INITRD="/var/tmp/ci-initramfs-$(uname -r).img"
 # command line arguments the original initrd was built with)
 cp -fv "/boot/initramfs-$(uname -r).img" "$INITRD"
 # Rebuild the original initrd without the multipath module
-dracut -o multipath --rebuild "$INITRD"
+dracut -a crypt -o "multipath rngd" --rebuild "$INITRD"
 
 # Initialize the 'base' image (default.img) on which the other images are based
 exectask "setup-the-base-image" "make -C test/TEST-01-BASIC clean setup TESTDIR=/var/tmp/systemd-test-TEST-01-BASIC"
 
+# Shared test env variables
+#
+# Explicitly set paths to initramfs and kernel images (for QEMU tests)
+# See $INITRD above
+export KERNEL_BIN="/boot/vmlinuz-$(uname -r)"
+# Explicitly enable user namespaces
+export KERNEL_APPEND="user_namespace.enable=1 ${CGROUP_KERNEL_ARGS[*]}"
+# Set timeouts for QEMU and nspawn tests to kill them in case they get stuck
+export QEMU_TIMEOUT=1800
+export NSPAWN_TIMEOUT=600
+# Work around 'Fatal glibc error: CPU does not support x86-64-v2'
+# See: https://access.redhat.com/solutions/6833751
+export QEMU_OPTIONS="-cpu type=q35,accel=tcg -cpu max"
```

Comment:
Note for future me: fix this embarrassing typo:

 ```suggestion
export QEMU_OPTIONS="-machine type=q35,accel=tcg -cpu max"
```

Reply:
Actually, drop the machine part completely and keep only the `-cpu max` stuff, as it seems to cause issues (at least on C8S).